cmd/lakectl/cmd/local.go

```diff
@@ -67,9 +67,9 @@ func localDiff(ctx context.Context, client apigen.ClientWithResponsesInterface,
 		return local.ListRemote(ctx, client, remote, currentRemoteState)
 	})
 
-	includeDirectoryMarkers := needsDirectoryMarkers()
+	includeUnixPermissions := cfg.Experimental.Local.UnixPerm.Enabled
```

Comment:
Changed the param name since it made no real sense to pass two flag params for `DiffLocalWithHead`.
Any suggestions for better naming of `includeUnixPermissions` will be welcome.